halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/kubernetes/KubernetesEditAccountCommand.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.providers.kubernetes;
+
+import com.beust.jcommander.Parameter;
+import com.netflix.spinnaker.halyard.cli.command.v1.providers.AbstractEditAccountCommand;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.providers.kubernetes.DockerRegistryReference;
+import com.netflix.spinnaker.halyard.config.model.v1.providers.kubernetes.KubernetesAccount;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class KubernetesEditAccountCommand extends AbstractEditAccountCommand<KubernetesAccount> {
+  protected String getProviderName() {
+    return "kubernetes";
+  }
+
+  @Parameter(
+      names = "--context",
+      description = "The kubernetes context to be managed by Spinnaker. "
+          + "See http://kubernetes.io/docs/user-guide/kubeconfig-file/#context for more information."
+  )
+  private String context;
+
+  @Parameter(
+      names = "--clear-context",
+      description = "Removes the currently specified context, defaulting to 'current-context' in your kubeconfig."
```

Comment:
Is 'specified context' here referring to the same thing as the 'managed context' from the description of the `--context` parameter just above?